src/data/pnat/basic.lean

```diff
@@ -304,6 +304,7 @@ theorem dvd_iff'' {k n : ℕ+} : k ∣ n ↔ ∃ m, k * m = n :=
 theorem dvd_intro {k n : ℕ+} (m : ℕ+) (h : k * m = n) : k ∣ n :=
  dvd_iff''.mpr ⟨m, h⟩
 
+@[simp]
```

Comment:
```suggestion
@[simp, refl]
```
@digama0 That's ok right, to tag this as `refl`.

Reply:
Tagging this with refl causes proofs in other files to break, when they try to implicitly use nat.dvd_refl I think.